### PR TITLE
Added ancestorScroll prop that takes an element to use as the scroll target

### PR DESCRIPTION
**Pull request by nero120**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default class InfiniteScroll extends Component {
   }
 
   componentDidMount () {
-    this.el = this.props.height ? this._infScroll : window;
+    this.el = this.props.height ? this._infScroll : this.props.ancestorScroll ? this.props.ancestorScroll : window;
```
Comment (ankeetmaini) [inline app/index.js]: can you please rename this to `scrollableTarget` (I feel this to be more generic, but I've no problems with ancestorScroll also). Also we can remove the ternary using
```js
this.el = this.props.height ? this._infScroll : this.props.scrollableTarget || window;
```

**Comment by ankeetmaini** [comment]
Also could you remove the file changes under demo and lib. Sorry for getting to it late! This looks great! Thanks so much for taking the time to send a PR!

**Comment by nero120** [comment]
All done @ankeetmaini!

**Comment by ankeetmaini** [comment]
This is awesome @nero120, could you also add documentation in README too, please? :)

**Comment by nero120** [comment]
README updated @ankeetmaini.

**Comment by ankeetmaini** [comment]
Thanks @nero120. Merged and released a major version with this change :)

**Comment by nero120** [comment]
Great stuff @ankeetmaini, thank you!

**Pull request merged**